src/main/java/org/openstreetmap/atlas/geography/atlas/complete/CompleteEdge.java

```diff
@@ -166,6 +175,36 @@ public int hashCode()
         return super.hashCode();
     }
 
+    @Override
+    public boolean isFull()
+    {
+        if (this.bounds == null)
```

Comment:
Coundn't this be one single statement with negated tests and ANDs?

Reply:
It could I suppose. I figured this might be easier to read. Sometimes the formatting makes if statements with lots of `&&` terms look really ugly.